python/libcudf/libcudf/load.py

```diff
@@ -47,9 +47,15 @@ def _load_wheel_installation(soname: str):
 def load_library():
     """Dynamically load libcudf.so and its dependencies"""
     try:
-        # libkvikio must be loaded before libcudf because libcudf references its symbols
+        # librmm and libkvikio must be loaded before libcudf because libcudf references
+        # them. librmm does not yet have a load_library() function, so we just import
+        # it.
```

Comment:
Comment is outdated:
```suggestion
        # them.
```

Reply:
Do we also want to mention rapids_logger on the previous line of the comment?